app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -28,9 +28,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("VisualVerificationActivity");
     private static final int SECURITY_CODE_LENGTH = 5;
+    private static final int NUM_BUTTONS = 12;
```

Comment:
Users can see how many tiles there are anyway. For example, on my emulator I see the button flash when I click on it, very subtle, but I can see that there's a button I'm clicking on and where it is / how large it is